beacon-chain/rpc/eth/validator/handlers_test.go

```diff
@@ -974,97 +962,13 @@ func TestGetAttestationData(t *testing.T) {
 		assert.Equal(t, http.StatusOK, writer.Code)
 	})
 
-	t.Run("handles in progress request", func(t *testing.T) {
```

Comment:
why are these tests removed?

Reply:
The notion of in-progress was part of the attestation data cache's mark-in-progress function, now that attestation data is removed and no longer used, this test becomes irrelevant